### PR TITLE
Add UsageMetadata to GenerateContentResponse

### PR DESCRIPTION
**Pull request by DellaBitta**
Adds the UsageMetadata payload to GenerateContentResponse.

- [X] Tests pass locally.
- [x] Tests pass
- [x] Appropriate changes to documentation are included in the PR

### CODE REVIEW

File: packages/main/types/responses.ts [hunk 1]
```diff
@@ -79,8 +79,28 @@ export interface EnhancedGenerateContentResponse
  * @public
  */
 export interface GenerateContentResponse {
+  /** Candidate responses from the model. */
   candidates?: GenerateContentCandidate[];
+  /**
+   * A set of the feedback metadata the prompt specified in
```
Comment (hsubox76) [inline packages/main/types/responses.ts:85]: I know this is from the proto, but it's kind of garbled, maybe try using the proto comment next to the property which I think is more clear: https://source.corp.google.com/piper///depot/google3/google/ai/generativelanguage/v1main/generative_service.proto;l=444

**Review by hsubox76 — COMMENTED** [review]
Thanks for remembering to do the docs, forgot to check that on the last PR!
Can you also add a changeset? Minor since it changes the API.

**Review by hsubox76 — COMMENTED** [review]
Looks good, can you just add a changeset (minor)?

**Comment by DellaBitta** [comment]
Whoops, yep!

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**